### PR TITLE
Heroku internet deployment preparations

### PR DESCRIPTION
**Pull request by Archanial**
So far pushing it from console works.
Docker DB integration and CI next.

Merging to master because CI will compile and deploy master branch.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,19 @@
 FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
 WORKDIR /src #/app
-EXPOSE 80
-EXPOSE 443
```
Comment (Archanial) [inline Dockerfile:4]: Removed because heroku sets ports automatically

File: src/Patronage.Api/Program.cs [hunk 2]
```diff
@@ -75,14 +73,20 @@ void SendData(IHost app)
 
 
     // Configure the HTTP request pipeline.
-    if (app.Environment.IsDevelopment())
+    /*if (app.Environment.IsDevelopment())
```
Comment (mateuszgiedzinski) [inline src/Patronage.Api/Program.cs:76]: Maybe we can pass some Env variable to set environment as Developement as in visual studio we set. 
Also if we must make changes like that do not left commented code. Just delete it. We have history in git repository.

File: src/Patronage.Api/Program.cs [hunk 3]
```diff
@@ -93,8 +97,15 @@ void SendData(IHost app)
 
     app.MapControllers();
 
+
+
     app.Run();
```
Comment (mateuszgiedzinski) [inline src/Patronage.Api/Program.cs:102]: I wonder if the code will be here all the time until it stops running and not go to loging port.

File: src/Patronage.Api/Program.cs (hunk 2 above)
Comment (Archanial) [reply src/Patronage.Api/Program.cs:70]: I guess we can set ENV variable in heroku and read it. I'll think of something

**Pull request merged**